ocaml/testsuite/tests/typing-layouts/modules_alpha.ml

```diff
@@ -351,7 +351,7 @@ Error: This type M4'.s should be an instance of type ('a : void)
        The layout of M4'.s is immediate, because
          of the annotation on the declaration of the type t.
```

Comment:
I'm surprised here. I would expect a reference to the definition of `s`, above. I think there may be a call to generalize that you're missing.

Reply:
Yes, that's unfortunate (that is, the hiding of more information). But it's internally consistent. And I claim that the right way to tell the reason for the layout of a well-typed definition is to use merlin or some other editor integration. So: OK.